crates/cli-support/src/js/mod.rs

```diff
@@ -118,6 +118,58 @@ const INITIAL_HEAP_VALUES: &[&str] = &["undefined", "null", "true", "false"];
 // Must be kept in sync with `src/lib.rs` of the `wasm-bindgen` crate
 const INITIAL_HEAP_OFFSET: usize = 32;
 
+lazy_static! {
+    static ref JS_KEYWORDS: HashSet<&'static str> = {
+        let mut keywords = HashSet::new();
+        keywords.insert("await");
+        keywords.insert("break");
+        keywords.insert("case");
+        keywords.insert("catch");
+        keywords.insert("class");
+        keywords.insert("const");
+        keywords.insert("continue");
+        keywords.insert("debugger");
+        keywords.insert("default");
+        keywords.insert("delete");
+        keywords.insert("do");
+        keywords.insert("else");
+        keywords.insert("enum");
+        keywords.insert("export");
+        keywords.insert("extends");
+        keywords.insert("false");
```

Comment:
Missing `eval`

Reply:
Is it a reserved word? I think it's just a built-in function and I'm not sure we want to have them in this list.